.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,19 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
 1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
 1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
+</details>
+
+<details>
+<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. `ssh <USER.NAME>@hasta.scilifelab.se`
+1. Book your testing time using the Pax software. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `hasta`. `us; paxa -u <user> -s hasta -r scout-stage`.
+1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
+1. Stop the service with current deployed branch: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b<this_branch>`
```

Comment:
```suggestion
1. Stop the service with current deployed branch: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
```

Reply:
Hmm, I thought I did that already? 😊

Reply:
Is it correct to say `stop the service` here or just deploy your PR branch or similar?